runtime/android/core_internal/src/org/xwalk/core/internal/XWalkCookieManagerInternal.java

```diff
@@ -132,6 +148,16 @@ public void setAcceptFileSchemeCookies(boolean accept) {
         nativeSetAcceptFileSchemeCookies(accept);
     }
 
+    private static String judgeUrl(String url) throws MalformedURLException {
+        URL newUrl = null;
+        try{
+            newUrl = new URL(url);
+        }catch (MalformedURLException e){
+            throw new MalformedURLException("Invalid URL!");
+        }
+        return newUrl.toString();
+    }
+
```

Comment:
If this method is only used by `getCookie` and just has this simple functionality, I don't see the necessity to make it a standalone method instead of adding code in `getCookie` directly.